packages/windows/src/RPA/Windows/keywords/locators.py

```diff
@@ -70,6 +79,23 @@ def __init__(self, item, locator):
         self.xcenter = rect.xcenter() if rect else -1
         self.ycenter = rect.ycenter() if rect else -1
 
+    def is_sibling(self, win_elem: "WindowsElement") -> bool:
+        """Returns `True` if the provided window element is a sibling."""
+        locator: Optional[Locator] = win_elem.locator
+        while locator:
+            if isinstance(locator, WindowsElement):
+                locator = locator.locator
+        if not locator:
```

Comment:
Can you use here the rare while else syntax? :)
while locator:
    Do stuff
else:
    Nothing to do

Reply:
That was actually a bug where I missed this:
```python
            else:  # reached a string/`None`
                break
```
which created an infinite loop.

Now that I added this, there might be cases where the initial locator is a `WindowsElement` and the accessed inner locator is `None` so I still need to check both this case (and the one where the initial locator is `None`) within a single checkpoint as of:
```python
        if not locator:
            return True  # nothing to check, can be considered sibling
```